ibc_test.go

```diff
@@ -296,3 +296,44 @@ func TestAnalyzeCode(t *testing.T) {
 	require.NoError(t, err)
 	require.True(t, report2.HasIBCEntryPoints)
 }
+
+func TestIBCMsgGetChannel(t *testing.T) {
+	const CHANNEL_ID = "channel-432"
+
+	msg1 := api.MockIBCChannelOpenInit(CHANNEL_ID, types.Ordered, "random-garbage")
+	msg2 := api.MockIBCChannelOpenTry(CHANNEL_ID, types.Ordered, "random-garbage")
+	msg3 := api.MockIBCChannelConnectAck(CHANNEL_ID, types.Ordered, "random-garbage")
+	msg4 := api.MockIBCChannelConnectConfirm(CHANNEL_ID, types.Ordered, "random-garbage")
+	msg5 := api.MockIBCChannelCloseInit(CHANNEL_ID, types.Ordered, "random-garbage")
+	msg6 := api.MockIBCChannelCloseConfirm(CHANNEL_ID, types.Ordered, "random-garbage")
+
+	require.Equal(t, msg1.GetChannel(), msg2.GetChannel())
+	require.Equal(t, msg1.GetChannel(), msg3.GetChannel())
+	require.Equal(t, msg1.GetChannel(), msg4.GetChannel())
+	require.Equal(t, msg1.GetChannel(), msg5.GetChannel())
+	require.Equal(t, msg1.GetChannel(), msg6.GetChannel())
+	require.Equal(t, msg1.GetChannel().Endpoint.ChannelID, CHANNEL_ID)
+}
+
+func TestIBCMsgGetCounterVersion(t *testing.T) {
+	const CHANNEL_ID = "channel-432"
+	const VERSION = "random-garbage"
+
+	msg1 := api.MockIBCChannelOpenInit(CHANNEL_ID, types.Ordered, VERSION)
+	v, ok := msg1.GetCounterVersion()
```

Comment:
That definitely looks more idiomatic go than returning `*string`.
I know it is annoying from Rust, but I think this is proper